flowermd/base/simulation.py

```diff
@@ -604,6 +601,8 @@ def run_update_volume(
 
         Parameters
         ----------
+        final_box_lengths : np.ndarray, shape=(3,), dtype=float, default None
```

Comment:
Does `final_box_lengths` has to be a unyt_array or it could be a unyt_array and numpy array?

Reply:
It should work with either, I'll update the doc strings to show both.